### ISSUE TITLE
Close button not working in camera page

### ISSUE THREAD

**Issue by sherbaev** [dimelo/Dimelo-Android#62]
Close button not working in camera page Pixel 7 Pro 14 beta, here is the screenrecord 
https://github.com/ringcentral/engage-digital-messaging-android/assets/31945873/d87757bb-e392-49fd-8f4c-2485aaeaf3c0

**Comment by DrissTM** [comment]
Hi @sherbaev, do you have the same issue on the Android 14 release version?

**Comment by sherbaev** [comment]
@DrissTM No, I could not reproduce it on Pixel 7 Android 14 release version

**Comment by DrissTM** [comment]
thanks for the quick answer.

Then it seems that the issue comes from the Android 14 beta and not from the Engage Messaging SDK, I'll close the issue but feel free to open a new one if you encounter this in a release version

**Comment by sherbaev** [comment]
Hello @DrissTM we are able to reproduce the issue on Pixel 7 Pro Android 14, Can you please help ?

**Comment by DrissTM** [comment]
Hello,

You confirm that you are able to reproduce with a public release version Android 14, right?

Do you see any logs that could be helpful to us?

**Comment by sherbaev** [comment]
Hello @DrissTM maybe these might be somehow useful ? 

> Chat.lambda$onResume$22
Fatal Exception: java.lang.NullPointerException
Attempt to invoke virtual method 'androidx.fragment.app.FragmentManager androidx.fragment.app.FragmentActivity.getSupportFragmentManager()' on a null object reference

![image](https://github.com/ringcentral/engage-digital-messaging-android/assets/31945873/0a78cc94-32e7-4f24-b678-3a0ab5274c07)
![image](https://github.com/ringcentral/engage-digital-messaging-android/assets/31945873/0c9ab69b-20da-4951-ad84-9cde84985578)
![image](https://github.com/ringcentral/engage-digital-messaging-android/assets/31945873/fecee23d-65df-4007-bf0b-73cbc60edcee)

**Comment by DrissTM** [comment]
Thanks @sherbaev, could you please also provide the version of our SDK you are currently using?

**Comment by sherbaev** [comment]
@DrissTM latest one 3.0.1

**Comment by sherbaev** [comment]
Hello @DrissTM any updates on this issue ?
